src/test/scala/org/broadinstitute/dsde/workbench/sam/matchers/AnyOfMatcher.scala

```diff
@@ -3,5 +3,25 @@ package org.broadinstitute.dsde.workbench.sam.matchers
 import org.mockito.ArgumentMatcher
 
 class AnyOfMatcher[T](expectedValues: Iterable[T]) extends ArgumentMatcher[T] {
+
+  /** Use in conjunction with argThat.
+    *
+    * Example usage:
+    * ```
+    * import org.mockito.Mockito._
+    * import org.mockito.ArgumentMatchers._
+    *
+    * val mockObj = mock(classOf[MyClass])
+    *
+    * when(mockObj.myMethod(argThat(new AnyOfMatcher(Seq("foo", "bar", "baz")))))
```

Comment:
I _think_ if we add an object to the bottom of this file like:

```
object MatchesOneOf {
  def apply[T](possibleValues: Iterable[T]): AnyOfMatcher[T] = new AnyOfMatcher[T](possibleValues)
}
```

Then we can have tests that read slightly clearer like:  

```
when(mockObj.myMethod(argThat(MatchesOneOf(Seq("foo", "bar", "baz")))))
    .thenReturn("matched") mockObj.myMethod("bar")
verify(mockObj).myMethod(argThat(MatchesOneOf(Seq("foo", "bar", "baz"))))
```
It's just a little extra piece to maintain but I think it makes it read ever so slightly easier which should make the tests easier to read and maintain.  

There might be some other ways we could fiddle with this too in order to make it a little easier to read, but you'd have to fiddle around with it to see what you can make work.

Reply:
Renamed `AnyOfMatcher` class to `MatchesOneOf`. Added companion object `MatchesOneOf`.